app/models/miq_template.rb

```diff
@@ -32,6 +32,10 @@ def self.eligible_for_provisioning
     where(:type => subclasses_supporting(:provisioning).map(&:name)).active
   end
 
+  def self.filter_with_name(name)
+    where("vms.name ILIKE ?", "%#{name}%")
```

Comment:
sorry to be that guy...

Can we do:

```suggestion
    where(arel_table[:name].matches("%#{name}%"))
```

This is actually defaulting the second 2 parameters `("%#{name}%", nil, false)` -- the last one being that false

see also: https://rubydoc.info/docs/rails/6.0.2.1/Arel/Nodes/Matches

Reply:
Hey @kbrock , I had made the change..